index.bs

```diff
@@ -418,7 +418,7 @@ err on the side of favoring consistency with that.
 
 <div class="example">
 One example is
-<a href="https://www.w3.org/TR/css-color-4/#funcdef-lab">Lab colors</a>:
+<a href="https://www.w3.org/TR/css-color-4/#funcdef-lab">CIE L*a*b* colors</a>:
```

Comment:
As [noted on Wikipedia](https://en.wikipedia.org/wiki/CIELAB_color_space), «_Referring to CIELAB as "Lab" without asterisks should be avoided to prevent confusion with [Hunter Lab](https://en.wikipedia.org/wiki/Hunter_Lab)_». "**CIE L\*a\*b\***" matches identification in [CSS Color Module Level 4](https://www.w3.org/TR/css-color-4/#lab-colors).